src/api/CalculationService.ts

```diff
@@ -278,6 +282,35 @@ export class CalculationService {
     return clause.final === "NA";
   }
 
+  mapMeasureGroup(group: Group): GroupPopulation {
```

Comment:
I think eventually we can remove mapMeasureGroup from edit testcase component?

Reply:
yeah that's the plan in a follow-up PR. Also to add test coverage for this function